plugins/content/adaptiveimage/adaptiveimage.php

```diff
@@ -11,6 +11,7 @@
 
 use Joomla\CMS\Plugin\CMSPlugin;
 use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\Component\Media\Administrator\AdaptiveImage\FocusStoreInterface;
 use Joomla\Component\Media\Administrator\AdaptiveImage\JSONFocusStore;
```

Comment:
`FocusStoreInterface` and `JSONFocusStore` are not Administrator specific.

Reply:
Then, ```AdaptiveImage``` has to be moved from ```/Administrator/Component/com_media/``` to ```/Component/com_media/```